src/ndi-source.cpp

```diff
@@ -893,17 +959,26 @@ void ndi_source_thread_start(ndi_source_t *s)
 	pthread_create(&s->av_thread, nullptr, ndi_source_thread, s);
 	blog(LOG_INFO,
 	     "[DistroAV] ndi_source_thread_start: '%s' Started A/V ndi_source_thread for NDI source '%s'",
-	     s->config.ndi_receiver_name.constData(),
-	     s->config.ndi_source_name.constData());
+	     obs_source_get_name(s->obs_source), s->config.ndi_source_name);
 }
 
 void ndi_source_thread_stop(ndi_source_t *s)
 {
 	if (s->running) {
 		s->running = false;
 		pthread_join(s->av_thread, NULL);
+		auto obs_source = s->obs_source;
+		auto obs_source_name = obs_source_get_name(obs_source);
+		blog(LOG_INFO,
+		     "[DistroAV] ndi_source_thread_stop: '%s' Stopped A/V ndi_source_thread for NDI source '%s'",
+		     obs_source_name, s->config.ndi_source_name);
 		if (!s->config.remember_last_frame) {
-			obs_source_output_video(s->obs_source, NULL);
+#if 1
+			blog(LOG_INFO,
+			     "[DistroAV] ndi_source_thread_stop: '%s' Audio Only: Forcing a blank/clean video frame",
```

Comment:
Actually this is just the wrong log message content.

"[DistroAV] ndi_source_thread_stop: '%s' Audio Only: Forcing a blank/clean video frame",

Actually this is just the wrong log message content.

"[DistroAV] ndi_source_thread_stop: '%s' : Forcing a blank/clean video frame",

(This here as nothing to do with Audio-only :D) 

Behavior : if the settings is to NOT keep the last frame. Wipe the texture when not visible anymore anywhere.

Reply:
Thanks for catching that.
I blame AI auto completion! :)